InsertToFile.cs

```diff
@@ -19,15 +19,15 @@ public static int GetValidInput()
         }
     }
 
-    public static void WriteToFile(string FileName, int i)
+    public static void WriteToFile(string fileName, int i)
     {
 
         try
         {
 
             (int id, string name, int grade) = Input.TakeStudentData(i);
 
-            using StreamWriter streamWriter = new(FileName, true);
```

Comment:
keep using statements on top of files, so that one statement works for all the related methods.

Reply:
It shows error the "fileName" is pulled from the function.